### DISCUSSION TITLE
Add multi level grouping to quick grid

### DISCUSSION THREAD

**Discussion by davidbuckleyni** [dotnet/aspnetcore#55566] [Ideas]
Grouping is a basic fundamental of any grid but what am taking about is multi level groups 

Say

 Drinks
   Soft Drinks
     Coke
     Fanta
   Warm Drinks 
     Tea
       Camameal Tea
       Green Tea
     COffee
       Black
       Normal 

Etc

 The grouping would have a plus or minus button to indicate if expanadable or collapsing 

the grouping model item can be any property declared on the model.

**Comment by markmouton22** [comment]
Hi :) @davidbuckleyni 

With a structure something like this ?

let menu = {
    "Drinks": {
        "Soft Drinks": ["Coke", "Fanta"],
        "Warm Drinks": {
            "Tea": ["Camomile Tea", "Green Tea"],
            "Coffee": ["Black", "Normal"]
        }
    }
};

**Reply by davidbuckleyni** [reply]
Hi yes something like tht be great but could be unlimted levels i guess
